install/infra/single-cluster/k3s/tools.tf

```diff
@@ -0,0 +1,28 @@
+module "certmanager" {
+  source = "../../modules/tools/cert-manager"
+
+  kubeconfig  = var.kubeconfig
+}
+
+module "externaldns" {
+  source      = "../../modules/tools/cloud-dns-external-dns"
+  kubeconfig  = var.kubeconfig
+  credentials = var.credentials
+  project     = var.project
+  region      = var.region
+  zone        = var.zone
```

Comment:
**Issue(blocker):** The cloud-dns-external-dns module doesn't define `region` and `zone` variables, can we omit these?

```suggestion
```

```
gitpod /workspace/gitpod/install/infra/single-cluster/k3s (nvn/k3s-tf-mod) $ make apply
╷
│ Warning: Backend configuration ignored
│ 
│   on ../../modules/k3s/main.tf line 11, in terraform:
│   11:   backend "gcs" {
│ 
│ Any selected backend applies to the entire configuration, so Terraform expects provider configurations only in the root module.
│ 
│ This is a warning rather than an error because it's sometimes convenient to temporarily call a root module as a child module for testing purposes, but this backend configuration block will have no effect.
╵
╷
│ Error: Unsupported argument
│ 
│   on tools.tf line 12, in module "externaldns":
│   12:   region      = var.region
│ 
│ An argument named "region" is not expected here.
╵
╷
│ Error: Unsupported argument
│ 
│   on tools.tf line 13, in module "externaldns":
│   13:   zone        = var.zone
│ 
│ An argument named "zone" is not expected here.
╵
make: *** [Makefile:101: apply-cluster] Error 1
```